src/Runner.Worker/StepsRunner.cs

```diff
@@ -356,7 +358,7 @@ private async Task RunStepAsync(IStep step, CancellationToken jobCancellationTok
                     Trace.Info($"Updated step result (continue on error)");
                 }
             }
-            Trace.Info($"Step result: {step.ExecutionContext.Result}");
+            Trace.Info($"Step result: {StringUtil.ConvertToJson(step.ExecutionContext.Result)}");
```

Comment:
Add formatting

Reply:
Do we need to do this? `Result` is a simple type (an enum), not a structure.